bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml

```diff
@@ -33,7 +33,7 @@
 		<item-type>Number:Temperature</item-type>
 		<label>Air Temp</label>
 		<description>Air Temp</description>
-		<state pattern="%1f %unit%" readOnly="true"/>
+		<state pattern="%1.0f%unit%" readOnly="true"/>
```

Comment:
Why did you remove the space between the value and the unit? Please add it again, we have that for all bindings. Same of the other occurrences.